test/common/router/BUILD

```diff
@@ -173,7 +173,17 @@ envoy_directory_genrule(
 filegroup(
     name = "route_corpus",
     testonly = 1,
-    srcs = [":corpus_from_config_impl"] + glob(["route_corpus/**"]),
+    srcs = select({
+        "ppc_build": [],
+        "//conditions:default": [":corpus_from_config_impl"],
+    }) + glob([
+        "route_corpus/**",
+    ]),
+)
+
+config_setting(
```

Comment:
Probably best to reuse the existing config_setting in https://github.com/envoyproxy/envoy/blob/master/bazel/BUILD. What would be best here is to add an `alias` there for `x86` builds, and a `TODO` here explaining that we assume host==target on x86 and need to later clean this up for cross-compile.

Reply:
https://docs.bazel.build/versions/master/be/general.html#cross-compilation-considerations is the best reference on this... I wonder if we move the `config_impl_test_static` from `srcs` to `tools` in https://github.com/envoyproxy/envoy/blob/238ffa94843965c0f80aea596b9f2a23479fb858/test/common/router/BUILD#L164 if that would make it "just work"..